sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -23,11 +21,12 @@
 
 
 def lambda_handler(event, context):
-    """Handler function for creating metadata, adding it to the payload,
-    and sending it to the opensearch instance.
+    """Handler function for creating metadata, adding it to the
+    database.
```

Comment:
```suggestion
    """Handler function for creating metadata, adding it to the database.

```
All one line for the header docstring.